src/mixins/DataStore/DataStore.js

```diff
@@ -197,7 +197,11 @@ export const DataStore = (
           self.page++;
         }
 
-        if (pageSize) self.pageSize = pageSize;
+        if (pageSize) {
+          self.pageSize = pageSize;
+        } else {
+          self.pageSize = getStoredPageSize("tasks", 30);
```

Comment:
minor tweak. I see that 30 is used on line 18 of the file and here, please set a constant variable and use that in stead of just putting 30, this way in the future if we want to change the default value from 30 to something else we do it in 1 spot